### DISCUSSION TITLE
Share panel, Inspect PanelJSON and Library panels for view role users

### DISCUSSION THREAD

**Discussion by baur** [grafana/grafana#68749] [Dashboards, Panels & Visualizations]
Can anyone explain to me: 
**Why for a user with the view role "Share panel, Inspect PanelJSON and Library panels"?**

![image](https://github.com/grafana/grafana/assets/2898361/21f5b968-07a2-4025-8422-9577de9ff836)
![image](https://github.com/grafana/grafana/assets/2898361/ee547863-b125-4b88-af92-7bb44692a2ae)
![image](https://github.com/grafana/grafana/assets/2898361/6e909436-108c-43fc-b200-3c1ffe8fcc2b)

**Comment by baur** [comment]
Especially when in PanelJSON you can see queries to DB

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This is phrased as a question, and we'd like questions about grafana to go to our[ community forum](https://community.grafana.com/). Feel free to make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a solution and would like to request the feature.
